api/package.json

```diff
@@ -18,18 +18,17 @@
   "dependencies": {
     "async": "^3.2.1",
     "bikram-sambat": "^1.7.0",
-    "bikram-sambat-bootstrap": "^1.5.0",
```

Comment:
Are we removing this from `api` because it's only needed by webapp?

Reply:
Yeah! I couldn't figure out why it was there, I presume by accident.